main.go

```diff
@@ -35,7 +35,7 @@ func readSettings() []string {
 
 	flag.BoolVar(&s.debug, "debug", false, "print debug output")
 
-	flag.BoolVar(&s.mlock, "mlock", true, "mlock mapped files in memory (only applies to cmd-line specified files).")
+	flag.BoolVar(&s.mlock, "mlock", true, "mlock mapped files in memory rather than copy to heap.")
```

Comment:
Would it be better to just have a flag for each one? Or specify by string a method?